images/ipfailover/keepalived/README.md

```diff
@@ -67,12 +67,12 @@ Pre-requisites/Prep Time
 
         $ vagrant ssh minion-1
         #  Add user and project.
-        openshift ex policy add-user view anypassword:test-admin
-        openshift ex new-project test --display-name="Failover Sample" \
-           --description="Router Failover" --admin=anypassword:test-admin
+        osadm policy add-role-to-user view test-admin
```

Comment:
it's not required.  I have no idea what the intent of this doc was.  It had rotted.